app/com/linkedin/drelephant/mapreduce/heuristics/GenericMemoryHeuristic.java

```diff
@@ -63,14 +64,21 @@ private void loadParameters() {
     logger.info(heuristicName + " will use " + MEM_RATIO_SEVERITY + " with the following threshold settings: "
         + Arrays.toString(memRatioLimits));
 
+    long containerMemDefaultBytes = CONTAINER_MEMORY_DEFAULT_BYTES;
+    if (paramMap.containsKey(CONTAINER_MEM_DEFAULT_MB)) {
+      containerMemDefaultBytes = Long.valueOf(paramMap.get(CONTAINER_MEM_DEFAULT_MB)) * FileUtils.ONE_MB;
+    }
+    logger.info(heuristicName + " will use " + CONTAINER_MEM_DEFAULT_MB + " with the following setting: "
```

Comment:
@shankar37 I agree with @akshayrai. Also, personally I don't find the wording confusing and I don't think having a second wording is worth the extra code complexity (even if it's minor).